bitnami/geode/templates/server/statefulset.yaml

```
@@ -286,15 +286,25 @@ spec:
           {{- if not .Values.diagnosticMode.enabled }}
           {{- if .Values.server.livenessProbe.enabled }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.server.livenessProbe "enabled") "context" $) | nindent 12 }}
-            tcpSocket:
-              port: server
+            exec:
+              command:
+                - sh
+                - -c
+                - |
+                  cd /bitnami/geode/data
+                  gfsh status server
```

Use bash + run the command in a sub-shell so changing directory scope is reduced:
```suggestion
                - bash
                - -ec
                - |
                  (cd /bitnami/geode/data && gfsh status server)
```